app/src/ui/list.tsx

```diff
@@ -136,6 +135,13 @@ interface IListProps {
   readonly onRowKeyDown?: (row: number, event: React.KeyboardEvent<any>) => void
 
   /**
+   * A handler called whenever a mouse down event is received on the
+   * row container element. Unlike onSelectionChanged, this is raised
+   * for every mouse down event, whether the row is selected or not.
+   */
+  readonly onRowMouseDown?: (row: number, event: React.MouseEvent<any>) => void
```

Comment:
Could we use https://github.com/desktop/desktop/blob/0c4a86ac4bad7b458aaad028e2420354a2b44137/app/src/ui/list.tsx#L665 instead of adding this? Entirely possible there's some nuance I've missed here.

Reply:
That's already wired up to insert a completion, but the underlying DOM event wasn't firing for me for this case and I couldn't understand why that's (not) happening.

Reply:
/shrug